src/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Button, FormControl, Input } from "@material-ui/core";
+import Service from "./services/getSimilar";
+const service = new Service();
+
+function App() {
+  
+  const [search, setSearch] = useState("");
+  const [result, setResult] = useState();
+
+  const handleChange = ({ target }) => setSearch(target.value);
```

Comment:
great way to deconstruct the event... you could take it a step further and do this

```javascript
const handleChange = ({ target: { value } }) => setSearch(value)
```

Reply:
oooh yeah! I will definitely consider it!